blog/docs/articles/consuming_s3_data_with_knative.md

```diff
@@ -0,0 +1,175 @@
+# Consuming AWS S3 Events with Knative Eventing, Functions and Apache Camel K
+
+**Author: Matthias Weßendorf, Senior Principal Software Engineer @ Red Hat**
+
+_In this blog post you will learn how to easily consume events from an AWS S3 bucker in a Knative Function with Knative Eventing and Functions. The blog post builds up on the first post of the series on Apache Camel K and Knative_
+
+
+One of the typical use-cases for a Serverless Function is to react to events delivered by an external source of events. A common example of this is to receive notifications from an AWS S3 bucket in a Serverless Function. But how to receive those events in an on premise environment, where the function does not run on AWS, but in a custom Kubernetes setup? 
+
+## Installation
+
+The [Installation](https://camel.apache.org/camel-k/next/installation/installation.html) from Apache Camel K offers a few choices, such as CLI, Kustomize, OLM or Helm. Example of Helm installation:
+
+```
+$ helm repo add camel-k https://apache.github.io/camel-k/charts/
+$ helm install my-camel-k camel-k/camel-k
+```
+
+Besides Camel K we also need to have Knative Eventing installed, as described in the [documentation](https://knative.dev/docs/install/yaml-install/eventing/install-eventing-with-yaml/){:target="_blank"}.
+
+## Creating a Knative Broker instance
+
+We are using a Knative Broker as the heart of our system, acting as an [Event Mesh](https://knative.dev/docs/eventing/event-mesh/){:target="_blank"} for both event producers and event consumers:
+
+```yaml
+apiVersion: eventing.knative.dev/v1
+kind: Broker
+metadata:
+  namespace: default
+  name: my-broker
+```
+
+Now event producers can send events to it and event consumers can receive events.
+
+## Using Kamelets as Event Sources for AWS S3
+
+In order to bind a Kamelet to a Knative component, like the above broker, we are using the `Pipe` API. A Pipe allows to declaratively move data from a system described by a Kamelet _towards_ a Knative destination **or** _from_ a Knative destination to another (external) system described by a Kamelet.
```

Comment:
```suggestion
In order to bind a Kamelet to a Knative component, like the broker we created above, we are using the `Pipe` API. A Pipe allows to declaratively move data from a system described by a Kamelet _towards_ a Knative destination **or** _from_ a Knative destination to another (external) system described by a Kamelet.
```
Saying "from a Knative destination to..." feels a little awkward to me, since the "destination" feels like it is where the event should be arriving, not coming from. But, I'm not sure what to replace it with

Reply:
I used resource.